src/components/cc-domain-management/cc-domain-management.smart.md

```diff
@@ -0,0 +1,68 @@
+---
+kind: '🛠 Domains/<cc-domain-management>'
+title: '💡 Smart'
+---
+
+# 💡 Smart `<cc-domain-management>`
+
+## ℹ️ Details
+
+<table>
+  <tr><td><strong>Component    </strong> <td><a href="https://www.clever-cloud.com/doc/clever-components/?path=/docs/🛠-domains-cc-domain-list--default-story"><code>&lt;cc-domain-management&gt;</code></a>
```

Comment:
nitpick: the link points to `cc-domain-list` instead of `cc-domain-management`

Reply:
nice catch, I would definitely have missed this one :roll_eyes: